client/js/components/fragments/ProfileInformations.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component, PropTypes } from 'react';
+
+class ProfileInformations extends Component {
+  render() {
+    const { user } = this.props;
+
+    return (
+      <div id="ProfileInformations" className="accordion list-group-item" role="tablist" aria-multiselectable="true">
+        <section className="panel panel-default">
+          <header href="#" className="panel-heading list-group-item-heading" role="tab" id="ProfileInformationsHeading">
+            <h4 className="panel-title" data-toggle="collapse" data-parent="#ProfileInformations"
+              aria-expanded="true" aria-controls="ProfileInformationsCollapse"
+              href="#ProfileInformationsCollapse">Informations...</h4>
```

Comment:
Could it just say "Information" instead of "Information..."? I'd also change the header, linked teams is information as well. I'd call it User or General. Or skip it at all, as this page already says "Profile page" a few pixels above.